onadata/libs/utils/cache_tools.py

```diff
@@ -26,6 +26,31 @@
 XFORM_LINKED_DATAVIEWS = "xfs-linked_dataviews"
 PROJECT_LINKED_DATAVIEWS = "ps-project-linked_dataviews"
 
+# Cache names used in the open_data viewet
+METADATA_FIELDS = [
+    '_review_comment',
+    '_review_status',
+    '_status',
+    '_edited',
+    '_version',
+    '_duration',
+    '_notes',
+    '_uuid',
+    '_tags',
+    '_bamboo_dataset_id',
+    '_attachments',
+    '_geolocation',
+    '_media_count',
+    '_total_media',
+    '_submitted_by',
+    'meta_instanceID',
```

Comment:
The `meta/instanceID` is part of the submitted data, I don't think we should remove it. `meta/deprecatedID` is usually also part of the data but only shows up if a record has been edited.

Reply:
Sure..seems consistent in both column data and row data from tests.